Core/Src/CAN/inverter_can_monitor.c

```diff
@@ -14,25 +14,35 @@
 
 void set_inverter_communication_error();
 void precharge_monitor();
+void left_inv_error_callback();
+void right_inv_error_callback();
+void inverter_can_diff(uint32_t id);
+
+static bool left_inv_received = 0;
+static bool right_inv_received = 0;
+static bool is_left_inv_active = 0;
+static bool is_right_inv_active = 0;
+static FDCAN_RxHeaderTypeDef RxHeader;
+
 
 void inverter_comm_error(void* argument) {
     UNUSED(argument);
 
     for (;;) {
         ECU_ENABLE_BREAKPOINT_DEBUG();
 
-        switch (osThreadFlagsWait(INVERTER_CAN_ACTIVE, osFlagsWaitAny,
-                                  INVERTER_NO_MESSAGE_ERROR_TIME)) {
+        uint32_t id = RxHeader.Identifier; //iniciei com esse valor pois está dando o erro " 'id' undeclared "
```

Comment:
Você precisar declarar a variável, mas não precisa inicializa-la.  
```suggestion
        uint32_t id;
```

Reply:
Vou falar isso aqui uma vez, mas serve para todos os casos que vc fez isso. Não é usado comentários com perguntas pro PR. Todo PR deve ser feito com o código pronto para ser feito o merge. E pergunta no comentário no código é ruim de ler, responder e  fácil de perder. Se precisar tira a dúvida antes ou então pode criar um comentário no github msm (tipo esse comentário).

Reply:
E passar clang-format antes de pedir avaliação do PR sempre tbm, pq o cube ide ferra a formatação e alguns lugares fica ruim de ler no github